src/pcapi/core/users/api.py

```diff
@@ -567,6 +567,11 @@ def create_pro_user(pro_user: ProUserCreationBodyModel) -> User:
     if pro_user.postal_code:
         new_pro_user.departementCode = PostalCode(pro_user.postal_code).get_departement_code()
 
+    if settings.IS_INTEGRATION:
```

Comment:
```suggestion
    if settings.IS_INTEGRATION is True:
```
Ce sera déjà un peu moins risqué :)

Reply:
@cgaunet : `IS_INTEGRATION` est un booléen. Qu'est-ce qu'apporterait l'ajout de `is True` ?